03_Modules/SmartCharging/src/module/module.ts

```diff
@@ -224,6 +280,12 @@ export class SmartChargingModule extends AbstractModule {
     props?: HandlerProperties,
   ): void {
     this._logger.debug('SetChargingProfile response received:', message, props);
+    const response: SetChargingProfileResponse = message.payload;
+    if (response.status === ChargingProfileStatusEnumType.Rejected) {
```

Comment:
Should we associate the response to the object we saved when we made the request?

Reply:
> Should we associate the response to the object we saved when we made the request?

There is no identifier in the SetChargingProfileResponse which enables us to find the corresponding stored object. There is a plan to find out a solution to match the response and request and refactor the current code. But since we are busy on other topics, this improvement hasn't been done.